### PR TITLE
Retirement of SSH over Local Companion in VS Code Desktop changelog

### PR DESCRIPTION
**Pull request by akosyakov**
## Description

Before merging we need to release a new version of VS Code extension.

## Related Issue(s)

Part of EXP-76

## Release QA Checklist ✔️

<details>
  <summary>QA Checklist </summary>

## [Priority: High] Functional Testing

-   [x] Test all links to make sure they are working correctly
-   [x] Verify that all forms (sales, support, docs feedback) are functioning properly and data is being captured correctly
-   [x] Check all buttons and calls-to-action to ensure they are leading to the correct pages
-   [x] Ensure that the website is mobile responsive and functions appropriately on different devices
-   [x] Please make sure that page is best for Light/Dark Mode. ⇒ Text/Media Changes

## [Priority: High] Content Testing

-   [x] Check all website copies for accuracy and spelling/grammar errors
-   [x] Verify that all images and videos are displaying properly
-   [x] Ensure that all downloadable files are working correctly
-   [x] Check that all external links are functioning properly

## [Priority: Medium] Performance Testing

-   [x] Try to use optimal media assets. ([Decision Record - How To](https://github.com/gitpod-io/website/blob/main/docs/adr/0003-optimal-assets-format.md))
-   [x] Test website load times to ensure that the website is loading quickly
-   [x] Verify that the website can handle high traffic volumes without crashing
-   [x] Check that all website functionality is working properly under heavy load
-   [x] Perform cross-browser testing to ensure that the website works properly on all major browsers (Google Chrome, Firefox, Edge)

## [Priority: Medium] Accessibility Testing

-   [x] Ensure that the website meets basic accessibility guidelines.
-   [x] Test that users with disabilities can easily navigate and use the website
-   [x] Verify that all website content can be accessed using assistive technologies

By including these key items in your website release QA checklist, you can ensure that your website is fully functional, user-friendly, and meets your business requirements.

## Security Testing

> **Note**:💡 If we are changing something related to cookies, data storing, or analytics.

-   [x] Verify that the website is HTTPS secure and that there are no security vulnerabilities
-   [x] Ensure that all sensitive data is being stored and transmitted securely

</details>

### CODE REVIEW

File: src/content/changelog/2023-10-05-local-app-retired-in-vscode-desktop.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+-   **Legacy Mode**: SSH Gateway will remain available as a fallback for now. SSH over Local Companion is being retired specifically for VS Code Desktop.
+
+-   **Local Companion**: The Local Companion feature will continue to be supported for use with VS Code Browser.
+
+## No Action Required
+
+If you've been using SSH over Local Companion, your setup will automatically switch to the optimized mode. For troubleshooting, consult our [guide](/docs/help/troubleshooting#gitpod-logs-in-vs-code-browser-and-desktop).
+
+We appreciate your ongoing support and look forward to enhancing your Gitpod experience! 🎉
```
Comment (gtsiolis) [inline src/content/changelog/2023-10-05-local-app-retired-in-vscode-desktop.md:21]: **thought**: Not sure about how to feel about this line. 🤷

File: src/content/changelog/2023-10-05-local-app-retired-in-vscode-desktop.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+-   **Streamlined Experience**: We're focusing on offering you a single, reliable mode for connecting VS Code Desktop with Gitpod. Details can be found in our [updated documentation](/docs/references/ides-and-editors/vscode#connecting-to-vs-code-desktop). Ensure you're using the latest versions of VS Code, Remote SSH, and the Gitpod extension for the most seamless experience.
+
+-   **Legacy Mode**: SSH Gateway will remain available as a fallback for now. SSH over Local Companion is being retired specifically for VS Code Desktop.
+
+-   **Local Companion**: The Local Companion feature will continue to be supported for use with VS Code Browser.
+
+## No Action Required
+
+If you've been using SSH over Local Companion, your setup will automatically switch to the optimized mode. For troubleshooting, consult our [guide](/docs/help/troubleshooting#gitpod-logs-in-vs-code-browser-and-desktop).
```
Comment (gtsiolis) [inline src/content/changelog/2023-10-05-local-app-retired-in-vscode-desktop.md:19]: **praise**: Nice help link!

File: src/content/changelog/2023-10-05-local-app-retired-in-vscode-desktop.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+title: Retirement of SSH over Local Companion in VS Code Desktop
+alt: Image banner for Retirement of SSH over Local Companion in VS Code Desktop
+date: 2023-10-05
+excerpt: We're streamlining your VS Code Desktop experience by retiring SSH over Local Companion.
+image: retirement-ssh-local-companion-in-vscode-dekstop.jpg
+ogImage: retirement-ssh-local-companion-in-vscode-dekstop.jpg
+customSlug: retirement-of-ssh-over-local-companion-in-vs-code-desktop
+---
+
+-   **Streamlined Experience**: We're focusing on offering you a single, reliable mode for connecting VS Code Desktop with Gitpod. Details can be found in our [updated documentation](/docs/references/ides-and-editors/vscode#connecting-to-vs-code-desktop). Ensure you're using the latest versions of VS Code, Remote SSH, and the Gitpod extension for the most seamless experience.
```
Comment (gtsiolis) [inline src/content/changelog/2023-10-05-local-app-retired-in-vscode-desktop.md:11]: **nitpick**: Does this need to be also in a list? I'd expect to read something, the main message, as a plain paragraph even if it's just 2-3 sentences.

**Review by gtsiolis — COMMENTED** [review]
Thanks, @akosyakov. 🌮 

Took a bias for action and replaced the image with an image that highlights the extension. If this is out of context or not related with the update, I'm adding the original image you had before as .webp in the [relevant discussion](https://gitpod.slack.com/archives/C045J13P8A1/p1696521752163889?thread_ts=1696518411.120399&cid=C045J13P8A1) (internal).

File: src/content/changelog/2023-10-05-local-app-retired-in-vscode-desktop.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+---
+title: Retirement of SSH over Local Companion in VS Code Desktop
```
[the remark this answers is not in the document]
Comment (akosyakov) [reply src/content/changelog/2023-10-05-local-app-retired-in-vscode-desktop.md:2]: This changelog to highlight that we don't support Local Companion mode anymore.

[the remark this answers is not in the document]
Comment (akosyakov) [reply src/content/changelog/2023-10-05-local-app-retired-in-vscode-desktop.md:2]: i.e we still do HTTP tunnel but differently

File: src/content/changelog/2023-10-05-local-app-retired-in-vscode-desktop.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
```
Comment (loujaybee) [inline src/content/changelog/2023-10-05-local-app-retired-in-vscode-desktop.md:2]: Suggestion: Might be better to use "deprecated" rather than "retirement"?

```suggestion
title: Local Companion (SSH) for VS Code Desktop is now deprecated
```

**Review by loujaybee — APPROVED** [review]

File: src/content/changelog/2023-10-05-local-app-retired-in-vscode-desktop.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+---
+title: Local Companion (as an SSH connection method) for VS Code Desktop is now retired
+alt: Image banner for Retirement of SSH over Local Companion in VS Code Desktop
+date: 2023-10-05
+excerpt: We're streamlining your VS Code Desktop experience by retiring SSH over Local Companion.
+image: retirement-ssh-local-companion-in-vscode-dekstop.webp
```
Comment (loujaybee) [inline src/content/changelog/2023-10-05-local-app-retired-in-vscode-desktop.md:6]: This image seems unrelated.

**Comment by loujaybee** [comment]
Just an FYI that [we are planning to release this change today](https://github.com/gitpod-io/website/pull/3988). We might want to roll this out Monday to avoid it getting lost in the noise. That said, I'm happy to announce this whenever, even today

**Comment by akosyakov** [comment]
@loujaybee My plan was to land it today in the evening with @jeanp413

File: src/content/changelog/2023-10-05-local-app-retired-in-vscode-desktop.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
+---
+title: Local Companion (as an SSH connection method) for VS Code Desktop is now retired
+alt: Image banner for Retirement of SSH over Local Companion in VS Code Desktop
+date: 2023-10-06
```
[the remark this answers is not in the document]
Comment (akosyakov) [reply src/content/changelog/2023-10-05-local-app-retired-in-vscode-desktop.md:4]: also in the name of a file

File: src/content/changelog/2023-10-05-local-app-retired-in-vscode-desktop.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+excerpt: We're streamlining your VS Code Desktop experience by retiring SSH over Local Companion.
+image: vscode.webp
+ogImage: vscode.webp
+customSlug: local-companion-ssh-connection-retired
+---
+
+Over the past few months we have continued to introduce more stable and reliable methods of connecting to workspaces via SSH. The previously supported method of establishing an SSH connection using the Local Companion app is now retired. No action is required by users, VS Code will automatically switch to the new connection method. For troubleshooting, consult our [guide](/docs/help/troubleshooting#gitpod-logs-in-vs-code-browser-and-desktop).
```
Comment (akosyakov) [inline src/content/changelog/2023-10-05-local-app-retired-in-vscode-desktop.md:11]: `/docs/references/ides-and-editors/vscode#troubleshooting` as a guide

**Comment by jeanp413** [comment]
Will merge on monday
